### PR TITLE
add facilmap

### PR DESCRIPTION
**Pull request by Strubbl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ This section is a great place to start if you want to get into improving OpenStr
 * [indoor=](https://indoorequal.org) - Map for viewing indoor data from OpenStreetMap.
 * [Qwant Maps](https://www.qwant.com/maps) - Modern map with POI search and routing functionality.
 * [F4map Demo](https://demo.f4map.com/) - 3D rendering demo of OpenStreetMap data.
+* [FacilMap](https://facilmap.org/) - Map with easy possibility to filter for POI types
```
Comment (Unkn0wnCat) [inline README.md:121]: Two small things about this:
1. FacilMap is Open Source, so consider adding the repo link
2. Descriptions need to end with a '.'

```suggestion
* [FacilMap](https://facilmap.org/) - Map with easy possibility to filter for POI types. ([Source Code](https://github.com/FacilMap/facilmap))
```

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
Great call - that map is pretty awesome, there are however two small changes I'd love to see before I merge this!

**Comment by Strubbl** [comment]
Thanks for keeping the list clean. I just applied your suggestion

**Review by Unkn0wnCat — APPROVED** [review]

**Comment by Unkn0wnCat** [comment]
Looking good! Thanks for your contribution 👍🏼

**Pull request merged**